config/Development.toml

```diff
@@ -43,7 +43,7 @@ locker_decryption_key2 = ""
 
```

Comment:
I have added the reviews for most of the files in this PR in L1-L3 PR (PR: #533 ).

Please attend to those changes.